From: 13832136087 <ningxiyuyi@126.com>
Date: Sun, 30 Aug 2026 07:23:01 +0000
Subject: =?utf-8?b?VHJhbnNsYXRlZCB1c2luZyBXZWJsYXRlICjmnJ3pspzor60gKGtvX0tQ?=
 =?utf-8?b?KSk=?=

Currently translated at 100.0% (49 of 49 strings)

Translation: openkylin-huanghe/libkysdk-applications
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/libkysdk-applications/ko_KP/
---
 kysdk-qtwidgets/translations/gui_ko_KP.ts | 64 +++++++++++++++----------------
 1 file changed, 32 insertions(+), 32 deletions(-)

diff --git a/kysdk-qtwidgets/translations/gui_ko_KP.ts b/kysdk-qtwidgets/translations/gui_ko_KP.ts
index 4477fad..f745c20 100644
--- a/kysdk-qtwidgets/translations/gui_ko_KP.ts
+++ b/kysdk-qtwidgets/translations/gui_ko_KP.ts
@@ -14,12 +14,12 @@
     <message>
         <location filename="../src/kcustomtabbar.cpp" line="518"/>
         <source>Scroll Left</source>
-        <translation>왼쪽으로 굴리기</translation>
+        <translation>왼쪽으로 스크롤</translation>
     </message>
     <message>
         <location filename="../src/kcustomtabbar.cpp" line="519"/>
         <source>Scroll Right</source>
-        <translation>오른쪽으로 굴리기</translation>
+        <translation>오른쪽으로 스크롤</translation>
     </message>
 </context>
 <context>
@@ -27,12 +27,12 @@
     <message>
         <location filename="../src/kaboutdialog.cpp" line="123"/>
         <source>Your system does not have any email application installed or the default mail application is not set up</source>
-        <translation>당신의 체계에는 어떠한 전자우편 응용 프로그람도 설치되지 않았거나 기본 메일 응용 프로그람이 설정되지 않았습니다</translation>
+        <translation>당신의 체계에는 어떠한 이메일 응용 프로그람도 설치되지 않았거나 기본 메일 응용 프로그람이 설정되지 않았습니다</translation>
     </message>
     <message>
         <location filename="../src/kaboutdialog.cpp" line="124"/>
         <source>Unable to open mail application</source>
-        <translation>이메일 클라이언트를 열 수 없습니다</translation>
+        <translation>메일 클라이언트를 열 수 없습니다</translation>
     </message>
     <message>
         <location filename="../src/kaboutdialog.cpp" line="258"/>
@@ -45,7 +45,7 @@
         <location filename="../src/kaboutdialog.cpp" line="264"/>
         <location filename="../src/kaboutdialog.cpp" line="278"/>
         <source>Privacy statement</source>
-        <translation>개인 정보 보호 정책</translation>
+        <translation>개인정보 보호 정책</translation>
     </message>
 </context>
 <context>
@@ -74,7 +74,7 @@
     <message>
         <location filename="../src/kdragwidget.cpp" line="103"/>
         <source>Select or drag and drop the folder identification path</source>
-        <translation>문서함을 선택하거나 끌어다 놓아 경로를 지정하십시오</translation>
+        <translation>폴더 선택 또는 드래그 앤 드롭하여 경로 지정</translation>
     </message>
 </context>
 <context>
@@ -98,7 +98,7 @@
     <message>
         <location filename="../src/kinputdialog.cpp" line="250"/>
         <source>ok</source>
-        <translation>확정</translation>
+        <translation>확인</translation>
     </message>
     <message>
         <location filename="../src/kinputdialog.cpp" line="252"/>
@@ -111,33 +111,33 @@
     <message>
         <location filename="../src/kmenubutton.cpp" line="72"/>
         <source>Options</source>
-        <translation>선택항</translation>
+        <translation>옵션</translation>
     </message>
     <message>
         <location filename="../src/kmenubutton.cpp" line="73"/>
         <source>Setting</source>
         <translatorcomment>设置</translatorcomment>
-        <translation>체계 설정</translation>
+        <translation>설정</translation>
     </message>
     <message>
         <location filename="../src/kmenubutton.cpp" line="74"/>
         <source>Theme</source>
-        <translation>테마</translation>
+        <translation>모양</translation>
     </message>
     <message>
         <location filename="../src/kmenubutton.cpp" line="75"/>
         <source>Help</source>
-        <translation>도움</translation>
+        <translation>도움말</translation>
     </message>
     <message>
         <location filename="../src/kmenubutton.cpp" line="76"/>
         <source>About</source>
-        <translation>본기 정보</translation>
+        <translation>관하여</translation>
     </message>
     <message>
         <location filename="../src/kmenubutton.cpp" line="77"/>
         <source>Feedback</source>
-        <translation>의견 반영하러 가기</translation>
+        <translation>의견 피드백</translation>
     </message>
     <message>
         <location filename="../src/kmenubutton.cpp" line="78"/>
@@ -147,17 +147,17 @@
     <message>
         <location filename="../src/kmenubutton.cpp" line="85"/>
         <source>Auto</source>
-        <translation>자동 검색</translation>
+        <translation>체계 따라가기</translation>
     </message>
     <message>
         <location filename="../src/kmenubutton.cpp" line="87"/>
         <source>Light</source>
-        <translation>가벼움</translation>
+        <translation>밝은 색</translation>
     </message>
     <message>
         <location filename="../src/kmenubutton.cpp" line="89"/>
         <source>Dark</source>
-        <translation>어두움</translation>
+        <translation>어두운 색</translation>
     </message>
 </context>
 <context>
@@ -174,7 +174,7 @@
         <location filename="../src/ksearchlineedit.cpp" line="662"/>
         <location filename="../src/ksearchlineedit.cpp" line="857"/>
         <source>Search</source>
-        <translation>전체 검색</translation>
+        <translation>검색</translation>
     </message>
 </context>
 <context>
@@ -182,17 +182,17 @@
     <message>
         <location filename="../src/ksecuritylevelbar.cpp" line="62"/>
         <source>Low</source>
-        <translation>약함</translation>
+        <translation>낮음</translation>
     </message>
     <message>
         <location filename="../src/ksecuritylevelbar.cpp" line="65"/>
         <source>Medium</source>
-        <translation>중</translation>
+        <translation>중간</translation>
     </message>
     <message>
         <location filename="../src/ksecuritylevelbar.cpp" line="68"/>
         <source>High</source>
-        <translation>고속</translation>
+        <translation>높음</translation>
     </message>
 </context>
 <context>
@@ -200,7 +200,7 @@
     <message>
         <location filename="../src/ksecuritylevelbar.cpp" line="179"/>
         <source>Low</source>
-        <translation>약함</translation>
+        <translation>낮음</translation>
     </message>
 </context>
 <context>
@@ -208,7 +208,7 @@
     <message>
         <location filename="../src/ksecurityquestiondialog.cpp" line="99"/>
         <source>Security question%1</source>
-        <translation>보안 문제%1</translation>
+        <translation>보안 질문%1</translation>
     </message>
     <message>
         <location filename="../src/ksecurityquestiondialog.cpp" line="106"/>
@@ -226,7 +226,7 @@
     <message>
         <location filename="../src/ksecurityquestiondialog.cpp" line="277"/>
         <source>Security Question</source>
-        <translation>안전 문제</translation>
+        <translation>보안 질문</translation>
     </message>
     <message>
         <location filename="../src/ksecurityquestiondialog.cpp" line="279"/>
@@ -236,12 +236,12 @@
     <message>
         <location filename="../src/ksecurityquestiondialog.cpp" line="285"/>
         <source>Cancel</source>
-        <translation>실행 취소</translation>
+        <translation>취소</translation>
     </message>
     <message>
         <location filename="../src/ksecurityquestiondialog.cpp" line="288"/>
         <source>Save</source>
-        <translation>확정</translation>
+        <translation>저장</translation>
     </message>
 </context>
 <context>
@@ -256,7 +256,7 @@
     <message>
         <location filename="../src/kuninstalldialog.cpp" line="185"/>
         <source>deb name:</source>
-        <translation>묶음 이름：</translation>
+        <translation>패키지 이름：</translation>
     </message>
     <message>
         <location filename="../src/kuninstalldialog.cpp" line="190"/>
@@ -269,7 +269,7 @@
     <message>
         <location filename="../src/kuninstalldialog.cpp" line="305"/>
         <source>deb name:</source>
-        <translation>묶음 이름：</translation>
+        <translation>패키지 이름：</translation>
     </message>
     <message>
         <location filename="../src/kuninstalldialog.cpp" line="309"/>
@@ -283,13 +283,13 @@
         <location filename="../src/kwindowbuttonbar.cpp" line="156"/>
         <location filename="../src/kwindowbuttonbar.cpp" line="178"/>
         <source>Maximize</source>
-        <translation>최대화하다</translation>
+        <translation>최대화</translation>
     </message>
     <message>
         <location filename="../src/kwindowbuttonbar.cpp" line="162"/>
         <location filename="../src/kwindowbuttonbar.cpp" line="183"/>
         <source>Restore</source>
-        <translation>열기</translation>
+        <translation>복원</translation>
     </message>
 </context>
 <context>
@@ -302,17 +302,17 @@
     <message>
         <location filename="../src/kwindowbuttonbar.cpp" line="280"/>
         <source>Close</source>
-        <translation>끄기</translation>
+        <translation>닫기</translation>
     </message>
     <message>
         <location filename="../src/kwindowbuttonbar.cpp" line="341"/>
         <source>Maximize</source>
-        <translation>최대화하다</translation>
+        <translation>최대화</translation>
     </message>
     <message>
         <location filename="../src/kwindowbuttonbar.cpp" line="346"/>
         <source>Restore</source>
-        <translation>열기</translation>
+        <translation>복원</translation>
     </message>
 </context>
 </TS>
